fbchat/client.py

```diff
@@ -2217,6 +2198,7 @@ def _parseMessage(self, content):
 
         for m in content["ms"]:
             mtype = m.get("type")
+            print(json.dumps(m, indent=4))
```

Comment:
You left some debug code there ;)

Reply:
Oh, sorry 😂 can this stay commented in origin code?